src/style/color.rs

```diff
@@ -420,11 +420,11 @@ mod tests {
 
     #[test]
     fn from_u32() {
-        assert_eq!(Color::from_u32(0x000000), Color::Rgb(0, 0, 0));
-        assert_eq!(Color::from_u32(0xFF0000), Color::Rgb(255, 0, 0));
-        assert_eq!(Color::from_u32(0x00FF00), Color::Rgb(0, 255, 0));
-        assert_eq!(Color::from_u32(0x0000FF), Color::Rgb(0, 0, 255));
-        assert_eq!(Color::from_u32(0xFFFFFF), Color::Rgb(255, 255, 255));
+        assert_eq!(Color::from_u32(0x0000_0000), Color::Rgb(0, 0, 0));
+        assert_eq!(Color::from_u32(0x00FF_0000), Color::Rgb(255, 0, 0));
+        assert_eq!(Color::from_u32(0x0000_FF00), Color::Rgb(0, 255, 0));
+        assert_eq!(Color::from_u32(0x0000_00FF), Color::Rgb(0, 0, 255));
+        assert_eq!(Color::from_u32(0x00FF_FFFF), Color::Rgb(255, 255, 255));
```

Comment:
This change looks a bit weird (as these values are hex representations of colors, and not strictly pure u32 values). Do you think it's worth adding an allow for this?